### DISCUSSION TITLE
How to set stop losses by percentage of portfolio?

### DISCUSSION THREAD

**Discussion by brndn16** [EA31337/EA31337#319] [Q&A] [answered]
Hi, I am new to this project, I just wanted to know if it is possible to set stop losses by percentage of portfolio, or just in general what have you found to work best for money management. 

Also, changing the strategies for various timeframes, is this recommended, just far to many variations to backtest that over a large data set.

**Comment by kenorb** [comment, marked as answer]
> Set stop losses by percentage of portfolio,

Yes, this can be added soon. Please follow GH-320 for more details.
As for the workaround, you can set EA's task with condition based on account equity (such as 1%), then to close all active trades when it's below it.

> ... changing the strategies for various timeframes ... too many variations to backtest that over a large data set

You can optimize strategies in parts, one timeframe at a time (less iterations). See: #295 for more details.

**Reply by brndn16** [reply]
Thank you for your response. I have also found in backtesting, If I set the lot size to anything other than auto (0) then no trades are taken. 

Have you experienced this with the latest release?

**Reply by kenorb** [reply]
Not as far as I know. You can create a new issue, I'm going to check.
